src/__tests__/HeadCollector.test.js

```diff
@@ -1,28 +1,9 @@
 import React from 'react';
-import { shallow, render } from 'enzyme';
+import { render } from 'enzyme';
 import HeadCollector from '../HeadCollector';
 import HeadTag from '../HeadTag';
 
 describe('HeadCollector', () => {
-  it('provides context object', () => {
-    const arr = [];
-    const wrapper = shallow(
-      <HeadCollector headTags={arr}>
-        <div />
-      </HeadCollector>
-    );
-
-    const contextObj = wrapper.instance().getChildContext().reactHeadTags;
-
-    // assert shape
-    expect(typeof contextObj).toBe('object');
-    expect(typeof contextObj.add).toBe('function');
-
-    // assert functionality
-    contextObj.add('test');
-    expect(arr).toEqual(['test']);
-  });
-
```

Comment:
Is there a comparable test we could add here?

Reply:
This is internal mechanism. I don't think it should be tested.